.github/deployments/helmfile.yaml

```diff
@@ -59,6 +59,10 @@ releases:
                 name: {{ requiredEnv "ACTION_SERVER_SECRET_NAME" }}
                 key: "ALGOLIA_DOCS_INDEX"
       - postgresql:
+          tag: "12.9.0"
+          extraEnv:
+            - name: PGDATA
```

Comment:
Why do we need this env var now?

Reply:
For the manual db migration to 12.9 it was required; I tried removing it subsequently but that makes it error out